segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListener.java

```diff
@@ -105,7 +110,17 @@
     @VisibleForTesting
     public PravegaConnectionListener(boolean enableTls, int port, StreamSegmentStore streamSegmentStore, TableStore tableStore) {
         this(enableTls, false, "localhost", port, streamSegmentStore, tableStore, SegmentStatsRecorder.noOp(), TableSegmentStatsRecorder.noOp(),
-                new PassingTokenVerifier(), null, null, true);
+                new PassingTokenVerifier(), null, null, true,
+                ExecutorServiceHelpers.newScheduledThreadPool(1, "test-token-expiry-handler"));
```

Comment:
Who manages this executor?

Reply:
Even though the preexisting defaultBuilder() method where this executor was added, was only meant for testing, it was inappropriate to add the executor here. So, I've removed it altogether in the latest push. Please review the revised code.